spvc/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_executable(spvc_exe src/main.cc)
+shaderc_default_compile_options(spvc_exe)
+target_include_directories(spvc_exe PRIVATE ${shaderc_SOURCE_DIR}/libshaderc/include ${spirv-tools_SOURCE_DIR}/include)
+set_target_properties(spvc_exe PROPERTIES OUTPUT_NAME spvc)
+target_link_libraries(spvc_exe PRIVATE shaderc_spvc shaderc_util)
+add_dependencies(spvc_exe build-version)
+
+shaderc_add_asciidoc(spvc_doc_README README)
+
+if(SHADERC_ENABLE_INSTALL)
```

Comment:
Does this need to be ${SHADERC_ENABLE_INSTALL} ?

Reply:
No, and this shorter form is used all over shaderc.
I see I've been inconsistent however - I'll fix that.

Reply:
SHADERC_ENABLE_INSTALL and ${SHADERC_ENABLE_INSTALL} mean different things in cmake. I think it has to do with caching and options? I ran into issues with switftshader where it needed to switch to the ${} form.

It may work correctly with shaderc by itself, but embedding it into other projects is where I ran into issues with swiftshader.

Reply:
Actually only "if(SHADERC_ENABLE_INSTALL)" appears in shaderc at present.  If that's wrong, file a bug.

Reply:
k